css/style.css

```diff
@@ -57,7 +57,7 @@ p {
 }
 
 .connect-icon path:hover {
-  fill: #EF5350; 
+  fill: #EF5350;
```

Comment:
I know this wasn't you, but it is 2016. We probably shouldn't be using CSS. Can you take a look at integrating a proper stylesheet system like SASS?